lib/resources/pix.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * @name Pix
+ * @description This module exposes functions
+ *              related to the `/pix` path.
+ *
+ * @module balanceOperations
+ **/
+import routes from '../routes'
+import request from '../request'
+
+const createKey = (opts, body) =>
+  request.post(opts, routes.pix.keys, body || {})
+
+const getKeys = opts =>
```

Comment:
```suggestion
const all = opts =>
```

sugestão de alterar para manter a consistencia com outros recursos já mapeados